scripts/ci/distcheck_nonabi.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+if [ "${CC#clang}" != "${CC}" ] ; then
+	export CXX="clang++"
+fi
+
+cd "$(dirname "$0")"/../..
+./bootstrap
+./configure \
+	--enable-user-guides --disable-abi-compat
```

Comment:
I'd prefer to see standard script extended, rather than having separate nonabi test script. Not to mention that in this case you configure main ODP instance without abi-compat, but the ODP which distcheck will actually check will be tested with default option (abi-compat). Please use `DISTCHECK__CONFIGURE_FLAGS`.